src/bin/cli.ts

```diff
@@ -1,112 +1,63 @@
-import * as Yargs from 'yargs'
-import yargs from 'yargs/yargs'
-import * as browser from '../uploaders/BrowserUploader'
+import commandLineArgs from 'command-line-args'
+import commandLineUsage from 'command-line-usage'
 import logger from '../Logger'
+import uploadBrowser from '../commands/UploadBrowserCommand'
 
-const commonOpts: Record<string, Yargs.Options> = {
-  apiKey: {
-    describe: 'your project\'s API key',
-    type: 'string',
-    demandOption: true
+const topLevelDefs = [
+  {
+    name: 'command',
+    defaultOption: true
   },
-  overwrite: {
-    describe: 'whether to replace existing source maps uploaded with the same version',
-    default: false,
-    type: 'boolean'
+  {
+    name: 'help',
+    alias: 'h',
+    type: Boolean,
+    description: 'show this message'
   },
-  projectRoot: {
-    describe: 'the root of your project',
-    default: process.cwd(),
-    type: 'string'
-  },
-  verbose: {
-    describe: 'show detailed logs',
-    default: false,
-    type: 'boolean'
+  {
+    name: 'version',
+    type: Boolean,
+    description: 'output the version of the CLI module'
   }
-}
+]
 
-export default function run (argv: string[]): void {
-  const program = yargs()
+export default async function run (argv: string[]): Promise<void> {
+  const opts = commandLineArgs(topLevelDefs, { argv, stopAtFirstUnknown: true })
```

Comment:
This can throw if you give the same option twice, e.g. `./bin/cli --help --help`, so I think we need a try/catch here